packages/sdk-core/src/model/report/ReportAttribute.ts

```diff
@@ -0,0 +1,6 @@
+import { AttributeType } from '../data/BacktraceData';
+
+export interface ReportAttribute {
```

Comment:
Can we name this `ReportData` or something similar? This contains not only attributes.